gflownet/gflownet.py

```diff
@@ -394,15 +400,15 @@ def sample_batch(
                 envs, actions, valids = self.forward_sample(
                     envs,
                     times,
-                    sampling_method="policy",
+                    sampling_method="mixt",
```

Comment:
you most likely do not want to have "mixt" sampling method as default when sampling candidates for test!